Scripts/update-translations.rb

```diff
@@ -124,27 +121,52 @@
   langs = ALL_LANGS
 end
 
+script_root = __dir__
+project_dir = File.dirname(script_root)
+
 langs.each do |code,local|
-  lang_dir = File.join('WordPress', 'Resources', "#{local}.lproj")
-  puts "Updating #{code}"
-  system "mkdir -p #{lang_dir}"
+  lang_dir = File.join(project_dir, 'WordPress', 'Resources', "#{local}.lproj")
+  puts "Updating #{code} in #{lang_dir}"
+  system "mkdir", "-p", lang_dir
 
-  # Download for production
-  system "if [ -e #{lang_dir}/Localizable#{strings_file_ext}.strings ]; then cp #{lang_dir}/Localizable#{strings_file_ext}.strings #{lang_dir}/Localizable#{strings_file_ext}.strings.bak; fi"
+  destination = "#{lang_dir}/Localizable#{strings_file_ext}.strings"
+  backup_destination = "#{destination}.bak"
+
+  # Step 2 – Download the new strings
+  if File.exist? destination
+    FileUtils.copy destination, backup_destination
+  end
 
   url = "#{download_url}/#{code}/default/export-translations?#{strings_filter}format=strings"
-  destination = "#{lang_dir}/Localizable#{strings_file_ext}.strings"
 
-  system "curl -fLso #{destination} \"#{url}\"" or begin
+  system "curl", "-fgsLo", destination, url or begin
     puts "Error downloading #{code}"
   end
 
-  if File.size(destination).to_f == 0
-    abort("\e[31mFatal Error: #{destination} appears to be empty. Exiting.\e[0m")
+  # Step 3 – Validate the new file
+  if !File.exist?(destination) or File.size(destination).to_f == 0
+    puts "\e[31mFatal Error: #{destination} appears to be empty. Exiting.\e[0m"
+    abort()
   end
 
-  system "./Scripts/fix-translation #{lang_dir}/Localizable#{strings_file_ext}.strings"
-  system "plutil -lint #{lang_dir}/Localizable#{strings_file_ext}.strings" and system "rm #{lang_dir}/Localizable#{strings_file_ext}.strings.bak"
-  system "grep -a '\\x00\\x20\\x00\\x22\\x00\\x22\\x00\\x3b$' #{lang_dir}/Localizable#{strings_file_ext}.strings"
+  fix_script_path = File.join(script_root, 'fix-translation')
+
+  # References a file like: "#{lang_dir}.Localizable-old.strings" where strings_file_ext == "old"
+  strings_file_path = File.join(lang_dir, "Localizable#{strings_file_ext}.strings")
+
+  system fix_script_path, strings_file_path
+  system "plutil", "-lint", strings_file_path
+  system "grep", "-a", "\\x00\\x20\\x00\\x22\\x00\\x22\\x00\\x3b$", strings_file_path
+
+  # Clean up after ourselves
+  FileUtils.rm_f backup_destination
+
+  # Delete the downloaded file if we're using a strings filter
+  if !strings_file_ext.empty?
+        FileUtils.rm_f destination
```

Comment:
nit: indentation inconsistency (too many spaces)
```suggestion
    FileUtils.rm_f destination
```

Reply:
I'm not sure I understand this 🤔 I _think_ we should never delete the `destination` file.
But I may be missing something here, tbh. Why are you doing this?

Reply:
Ah, thanks for following up on this. I thought this script would merge them, but it looks like that actually happens in `ios_merge_translators_strings`, so you're right – it shouldn't be removed. 

Fixed in 6f0d1894c27577a6d3ea8a6a53853b1f2fa169f0